package.json

```diff
@@ -33,6 +33,7 @@
     "moment-timezone": "0.5.23",
     "pdfjs-dist": "2.0.489",
     "perfect-scrollbar": "0.8.1",
+    "popper": "1.0.1",
```

Comment:
Is it this library? https://github.com/FezVrasta/popper.js/

Reply:
Yes, this is it. Sorry for the mistake. I updated the PR and changed this `popper` to the latest `popper.js` version.

Reply:
+1, and license of the repo looks good (uses MIT).